client/dashboard/me/billing-purchases/payment-method-image.tsx

```diff
@@ -50,5 +50,11 @@ export function getPaymentMethodImageURL( paymentMethodType: string ): string {
 }
 
 export function PaymentMethodImage( { paymentMethodType }: { paymentMethodType: string } ) {
-	return <img alt={ paymentMethodType } src={ getPaymentMethodImageURL( paymentMethodType ) } />;
+	return (
+		<img
+			className="payment-method-image"
+			alt={ paymentMethodType }
+			src={ getPaymentMethodImageURL( paymentMethodType ) }
```

Comment:
How about defining height/width here instead of CSS?

```suggestion
			src={ getPaymentMethodImageURL( paymentMethodType ) }
			height={ 19 }
			width={ 30 }
```

Reply:
Good idea!